frontend/src/views/AmendingLawMetadataEditor.vue

```diff
@@ -171,23 +177,21 @@ const elementLinks = computed(() => {
 
           <!-- Time boundary selection -->
           <div class="px-16 pb-20 pt-10">
-            <label for="timeBoundarySelect">
-              <span class="ris-label3-regular">Zeitgrenze</span>
+            <div class="flex flex-col gap-6">
+              <!-- eslint-disable-next-line vuejs-accessibility/label-has-for -->
+              <label id="timeBoundarySelect">
```

Comment:
Todo: should not use a hardcoded ID, use `useElementId` instead. See [styleguide](https://digitalservicebund.atlassian.net/wiki/spaces/VER/pages/884310020/Frontend+conventions+and+patterns#Avoid-using-IDs.-Do-not-assume-them-to-be-human-readable-or-static.)